### ISSUE TITLE
BombSquad is 70% fun as it was

### ISSUE THREAD

**Issue by MrPomajdor** [efroemling/ballistica#178]
It's 70% fun as it was because of "pro" restrictions. **NO CUSTOM PLAYLIST :(**

I was always playing on custom playlist with friends. Now what, I wont buy pro.

**Comment by SmokePuddingEveryday** [comment]
Yeah I don't think setting custom playlists behind Pro was a great idea.

**Comment by MrPomajdor** [comment]
@efroemling 

Maybe Eric will bring back custom playlist without pro...


"Bring it back, bring it back , bring it back"
-Freddie Mercury 1975

**Comment by Teals53** [comment]
just support the game dont worry its not kill you xD

**Comment by astrojade** [comment]
Why don't you just link with someone who has the pro version?

**Comment by MMatiasMG** [comment]
Well... after almost 4 years custom playlists are going to be free again

**Comment by MatiasMGz** [comment]
BS Pro has been deleted. EVERYTHING its FREE! Please close this.

**Comment by Teals53** [comment]
lmao

**Comment by MatiasMGz** [comment]
r/FuckBSPro
